### PR TITLE
Converted Models from Java to Kotlin on convertModelsBranch

### PR DESCRIPTION
**Pull request by adityapatel-sudo**
Changed Flashcard and Deck Models into Kotlin from Java.

### CODE REVIEW

File: app/src/main/java/com/adityap/flashy_createflashcards/FlashcardDatabaseHelper.kt [hunk 1]
```diff
@@ -67,10 +69,7 @@ class FlashcardDatabaseHelper  //private static final String FOREIGN_KEY_LINK =
         val flashcardList: MutableList<FlashcardModel> = ArrayList()
         if (cursor.moveToFirst()) {
             do {
-                val flashcard = FlashcardModel()
-                flashcard.setmId(cursor.getInt(cursor.getColumnIndex(DECK_ID)))
-                flashcard.word = cursor.getString(cursor.getColumnIndex(WORD))
-                flashcard.definition = cursor.getString(cursor.getColumnIndex(DEFINITION))
+                val flashcard = FlashcardModel(cursor.getInt(cursor.getColumnIndex(DECK_ID)), cursor.getString(cursor.getColumnIndex(WORD)), cursor.getString(cursor.getColumnIndex(DEFINITION)))
```
Comment (swatisaoji1) [inline app/src/main/java/com/adityap/flashy_createflashcards/FlashcardDatabaseHelper.kt]: can we align it in multiple lines for readability

File: app/src/main/java/com/adityap/flashy_createflashcards/models/DeckModel.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.adityap.flashy_createflashcards.models
+
+import android.os.Parcel
+import android.os.Parcelable
+
+/**
+ * Created by Aditya on 12/26/2018.
+ */
+data class DeckModel(val id : Int = 0, val deckName: String?, val deckDescription: String?) : Parcelable {
```
Comment (swatisaoji1) [inline app/src/main/java/com/adityap/flashy_createflashcards/models/DeckModel.kt]: can we have a null deckName , I feel we should not make it nullable

File: app/src/main/java/com/adityap/flashy_createflashcards/models/FlashcardModel.kt [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package com.adityap.flashy_createflashcards.models
+
+import android.os.Parcel
+import android.os.Parcelable
+
+/**
+ * Created by Aditya on 9/30/2018.
+ */
+data class FlashcardModel(val id: Int = 0, val word: String?, val definition: String?) : Parcelable {
```
Comment (swatisaoji1) [inline app/src/main/java/com/adityap/flashy_createflashcards/models/FlashcardModel.kt]: I think we need both work and definition , making it nullable wouldnt help.

File: app/src/main/java/com/adityap/flashy_createflashcards/models/FlashcardModel.kt [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    companion object CREATOR : Parcelable.Creator<FlashcardModel> {
+        override fun createFromParcel(parcel: Parcel): FlashcardModel {
+            return FlashcardModel(parcel)
+        }
+
+        override fun newArray(size: Int): Array<FlashcardModel?> {
+            return arrayOfNulls(size)
+        }
+    }
+}
```
Comment (swatisaoji1) [inline app/src/main/java/com/adityap/flashy_createflashcards/models/FlashcardModel.kt]: add a newline at the end of file

**Review by swatisaoji1 — CHANGES_REQUESTED** [review]
Some nits mostly

**Comment by adityapatel-sudo** [comment]
* Converted Models from Java to Kotlin on convertModelsBranch
Addressed:
-can we align it in multiple lines for readability
-can we have a null deckName , I feel we should not make it nullable
-I think we need both work and definition , making it nullable wouldnt help.
-add a newline at the end of file

**Pull request merged**